src/vs/workbench/contrib/positronNotebook/browser/notebookCells/CellEditorMonacoWidget.tsx

```diff
@@ -123,6 +123,27 @@ export function useCellEditorWidget(cell: PositronNotebookCellGeneral) {
 		};
 	}, [cell, environment, instance, services.configurationService, services.instantiationService, services.logService]);
 
+	// Watch for editor focus requests from the cell
+	React.useLayoutEffect(() => {
+		// Subscribe to focus request observable
+		const disposable = autorun(reader => {
+			const shouldFocus = cell.editorFocusRequested.read(reader);
+			if (shouldFocus) {
+				const editor = cell.editor;
+				if (editor) {
+					editor.focus();
+					// Reset the focus request flag after consuming it (one-shot signal)
```

Comment:
Can hopefully avoid this as well with `observableSignal`, otherwise maybe a good old `Event` is simpler?